site/docs/v1/tech/messengers/index.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: doc
+title: Messengers
+description: Adding and Configuring Messengers
+---
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Messengers
+
+Messengers are used to send messages from within FusionAuth to other systems. They can be customized using link:/docs/v1/tech/themes/[themes].
```

Comment:
I don't believe you can use themes to customize a message. You can customize them using variables and you can localize them, but you can't use themes.

I would drop the last sentence ("they can...") because message templates are customized, not messengers.

Am I missing something?

Reply:
You are not missing anything... 😄